gnome-initial-setup/gis-driver.c

```diff
@@ -934,6 +959,7 @@ gis_driver_init (GisDriver *driver)
 
   g_signal_connect (screen, "size-changed",
                     G_CALLBACK (screen_size_changed), driver);
+  g_signal_connect (driver, "activate", G_CALLBACK (emit_startup_sound), NULL);
```

Comment:
I don't think we want to do this at all on non-Hack images. There's code in `gis-summary-page.c` which checks for a Hack image:

```
if (gis_driver_is_product (GIS_PAGE (page)->driver, HACK_PRODUCT_NAME)
```

I suggest adding a `gis_driver_is_hack (GisDriver *)` method, changing the two call sites in `gis-summary-page.c` to use it, and using it here to make this conditional on being a hack image.